Tests/Tests.UWP/Tests.UWP.csproj

```diff
@@ -11,15 +11,14 @@
     <AssemblyName>Tests.UWP</AssemblyName>
```

Comment:
Reviewers can ignore changes to this file - since we don't run UWP tests automatically, the project file had rotted a little and needed to bring it up to date to be able to debug things.